packages/matchbox/src/components/Pagination/tests/Pagination.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Pagination from '../Pagination';
+
+describe('Pagination', () => {
+  let props;
+  let wrapper;
+
+  beforeEach(() => {
+    props = {
+      currentPage: 1,
```

Comment:
Will need to add a few more tests to cover all the conditionals in this component